From 1f9d18ecbc116e48852c76c6986d0d9b3a66b264 Mon Sep 17 00:00:00 2001
From: Shiyong Li <a22381@motorola.com>
Date: Wed, 6 Jan 2016 19:14:05 -0800
Subject: [PATCH 596/959] IKQCOMSD1-3106 arm/dts: 8952: make touch work-around
 not common

Use work-around power down sequence introduced by 4e04528 for
Affinity only. Other product touch does not have such issue.

Change-Id: I4846172d335d9d67f82da26267ea8c304e36b32a
Signed-off-by: Shiyong Li <a22381@motorola.com>
Reviewed-on: http://gerrit.mot.com/811230
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Konstantin Makariev <kmakariev@motorola.com>
Reviewed-by: Christopher Fries <cfries@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-athene.dtsi      | 9 +++++++++
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 3 +--
 2 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/msm8952-athene.dtsi b/arch/arm/boot/dts/qcom/msm8952-athene.dtsi
index dba36c856bc..55113c9facd 100644
--- a/arch/arm/boot/dts/qcom/msm8952-athene.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-athene.dtsi
@@ -49,3 +49,12 @@
 &ibb_regulator {
 	qcom,qpnp-ibb-discharge-resistor = <32>;
 };
+
+&i2c_3 { /* BLSP1 QUP3 */
+	synaptics@20 {
+		synaptics,use-in-progress-event-blank;
+		default_suspended: touchstate_aod_suspended {
+			patch-data = "F12@20:1=2;F1@0=1&7";
+		};
+	};
+};
diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index cd01ea0dfef..ae5fad0f790 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -908,7 +908,6 @@
 		gpios = <&msm_gpio 65 0>, /* IRQ */
 			<&msm_gpio 64 0>; /* RESET */
 		synaptics,touch-clip-area = <450 1875 610 1919>;
-		synaptics,use-in-progress-event-blank;
 		vdd_quirk-supply = <&pm8950_l6>;
 		synaptics,gpio-config;
 		/* pins used by touchscreen */
@@ -922,7 +921,7 @@
 			patch-data = "";
 		};
 		default_suspended: touchstate_aod_suspended {
-			patch-data = "F12@20:1=2;F1@0=1&7";
+			patch-data = "F1@0=1&7";
 		};
 	};
 };
-- 
2.11.0

